src/ukify/ukify.py

```diff
@@ -918,6 +935,117 @@ def make_uki(opts):
 
     print(f"Wrote {'signed' if sign_args_present else 'unsigned'} {opts.output}")
 
+def sign_uki(opts):
+    """
+    Unpack UKI, replace cmdline, measure and sign PCR, then repack and sign it.
+    We have to repack as PCR signature should be before the linux section.
+    """
```

Comment:
Nit: The first line of the docstring is currently empty. Also, let's add a few articles.
```suggestion
    """Unpack a UKI, replace the cmdline, measure and sign the PCR, then repack and sign the UKI.
    
    We have to repack the UKI because the PCR signature should be before the linux section.
    """
```